pydata_sphinx_theme/static/css/theme.css

```diff
@@ -0,0 +1,85 @@
+/*****************************************************************************
+ * Custom font
+**/
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+:root {
+  /*****************************************************************************
+   * Bootstrap variables are availlable:
+   * https://getbootstrap.com/docs/4.0/getting-started/theming/#css-variables
+  **/
+
+  /*****************************************************************************
+  * Font size
+  **/
+  --fs-base: 15px; /* base font size - applied at body / html level */
+
+  /* heading font sizes */
+  --fs-h1: 36px;
+  --fs-h2: 32px;
+  --fs-h3: 26px;
+  --fs-h4: 21px;
+  --fs-h5: 28px;
```

Comment:
this seems a typo? (28 doesn't really fit between 21 and 18 ;))

Reply:
yes indeed!